src/entry/index.js

```diff
@@ -0,0 +1,4 @@
+// @flow
+export { triggerLoadCoreEpic, loadAppEpic, loadCoreFailedEpic } from './entry.epics';
+export { actionTypes } from './entry.actions';
```

Comment:
I am gonna raise this argument here just to discuss it and see where we stand. It's basically out of the scope of this PR. 
I also know in the whole project we are exporting all the action types as `actionTypes` and when multiple of them are in the same file then we rename them as for example we do [here](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/reducers/descriptions/form.reducerDescription.js). 

My question is what if we would export them with a unique name so that we would be able to search around the project using their unique names? 
Now if I would like to search some types it might be difficult to find them. Another thing is that one `actionTypes` might have been renamed in one place differently than in an another one using `as`. I am worried that using different names might end up a little bit tricky in the long term. 

Again just food for thought. Let me know what you think

Reply:
Thanks for bringing this up. I've been swaying back and forth over this. I think I'm leaning the same direction as you in hindsight. Added the topic to our next Tracker technical discussion board meeting.

Reply:
Great stuff thanks a lot! Let me know if I could help with that :)